src/components/wallet/views/balanceDisplay.tsx

```diff
@@ -18,12 +10,15 @@ export default function balanceDisplay(this: Wallet) {
     }
     const isRegulated =
       !balance.is_authorized && balance.asset_type !== 'native'
+    const isNotTrusted = !balance.trusted && balance.asset_type !== 'native'
```

Comment:
I think you can just use `!balance.trusted` because you have XLM marked as `trusted: true`